### PR TITLE
Add rsync

### PR DESCRIPTION
**Pull request by enm10k**
Add rsync for `fabric.contrib.project.rsync_project`.
http://docs.fabfile.org/en/1.12/api/contrib/project.html#fabric.contrib.project.rsync_project

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,11 +2,9 @@ FROM alpine:3.4
 
 MAINTAINER Geshan Manandhar (http://geshan.com.np)
 
-RUN apk --update add curl &&\
-    apk add fabric --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted &&\
-    rm /var/cache/apk/*
+RUN apk add --no-cache curl rsync openssh &&\
+    apk add --no-cache fabric --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted
```
Comment (geshan) [inline Dockerfile:6]: please add back `rm /var/cache/apk/*` and rebase with latest master.

**Review by geshan — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (enm10k) [reply Dockerfile:6]: > please add back rm /var/cache/apk/* 

Thanks to `--no-cache` option, no need to `rm /var/cache/apk/*`.
You can check it like this.
`docker run -it cc01a56f105d ls /var/cache/apk`

> rebase with latest master.

Understood. I will ping you when it's finished.

Thank you.

Comment (enm10k) [reply Dockerfile:6]: @geshan Done. Please check it.

**Review by geshan — APPROVED** [review]
Merging

**Comment by enm10k** [comment]
Thanks.

**Comment by geshan** [comment]
Welcome

Warm Regards,
Geshan Manandhar

**Pull request merged**